src/components/molecules/Alert/Alert.tsx

```diff
@@ -61,7 +61,7 @@ export const Alert = ({
             aria-label={severity}
             role='alert'
             className={clsx('rounded-xl', {
-                'border-2 border-yellow bg-yellow/20': variant === 'outlined', // should be changed to handle different colors
+                'border-yellow bg-yellow/20 border-2': variant === 'outlined', // should be changed to handle different colors
```

Comment:
@kaushikriya Couple of things here:

1) Should the border be 1px instead?
2) Background color for this should be `bg-yellow-900/20` instead of `bg-yellow/20`

Reply:
Remind me because I might have forgotten: did we agree with Ben to use the old Alerts for this ticket? Meaning we would ignore the figma design for this Alert component.